examples/symbolic_logic/gries_schneider/test_gs.py

```diff
@@ -5,7 +5,9 @@
 from mathics.core.definitions import Definitions
 from mathics.core.evaluation import Evaluation
 from mathics.core.parser import MathicsSingleLineFeeder, parse
+from mathics.core.system_init import initialize_system
 
+initialize_system()
```

Comment:
This routine is called each time ```Definitions(add_builtins=True)``` is called. And what it essentially does is to initialize (a part of) the ```mathics.builtins``` module, so if you do not call it,  ```Definitions(add_builtins=True)``` can not do what it is expected. So, should not be better either
a) call it in ```mathics.buitlin.__init__``` module.
b) import it (locally) and call it  in ```mathics.core.definitions.Definitions.__init__```, if the `add_builtins==True`?

Reply:
I don't know how to resolve. I don't like a) because we want to _remove_ stuff from `mathics.builtin.__init__` - that is the whole _point_ of this. 

We want a function, `initialize_system()` that is explicitly called that will load will initialize all predefined Builtins. 

Once that is done, we are in a position to write a routine to dump out all of this to say a pickle file and then we have the possibility  to replace that code with code that reads in the dumped information instead. 

Equally important,  this is conceptually simpler, and coding-wise we don't have critical stuff or magic code inside the an  `__init__`  module.